src/shared/components/links/ExternalLink.tsx

```diff
@@ -34,35 +36,41 @@ const ExternalLink: React.FC<React.PropsWithChildren<ExternalLinkProps>> = ({
   icon,
   onClick,
   size = 'sm',
-}) => (
-  <AnalyticsButton
-    component="a"
-    style={style}
-    className={additionalClassName}
-    href={href}
-    target="_blank"
-    rel="noopener noreferrer"
-    data-test-id={dataTestID}
-    isInline={isInline}
-    variant={variant}
-    icon={icon}
-    onClick={(e) => {
-      if (stopPropagation) {
-        e.stopPropagation();
-      }
-      onClick?.(e);
-    }}
-  >
-    {children || text}
-    {!hideIcon ? (
-      <>
-        {' '}
-        <Icon iconSize={size}>
-          <ExternalLinkAltIcon />
-        </Icon>
-      </>
-    ) : null}
-  </AnalyticsButton>
-);
+  isHighlightable,
+}) => {
+  const combinedClassNames = `${additionalClassName} ${
+    isHighlightable ? 'highlightable-link' : ''
+  }`.trim();
```

Comment:
use css method from patternfly to apply classNames conditionally

`import { css } from '@patternfly/react-styles'; `

Reply:
It's usage is similar to `import classNames from 'classNames'`

Reply:
Thanks for the tip! Change has been added.